var/spack/repos/builtin/packages/xios/package.py

```diff
@@ -16,10 +16,10 @@ class Xios(Package):
 
     version("develop", svn="http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/trunk")
     version(
-        "2.5", revision=1860, svn="http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/branchs/xios-2.5"
+        "2.5", revision=2593, svn="http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/branchs/xios-2.5"
     )
     version(
-        "2.0", revision=1627, svn="http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/branchs/xios-2.0"
+        "2.0", revision=2593, svn="http://forge.ipsl.jussieu.fr/ioserver/svn/XIOS/branchs/xios-2.0"
```

Comment:
This doesn't make much sense to me. Why are you bumping the revision of two old versions?

Reply:
@alalazo 
I have reverted this change as it is not a necessary fix.